trino-connector/src/main/java/org/apache/gravitino/trino/connector/system/table/GravitinoSystemTableCatalog.java

```diff
@@ -56,21 +60,34 @@ public GravitinoSystemTableCatalog(CatalogConnectorManager catalogConnectorManag
 
   @Override
   public Page loadPageData() {
-    List<GravitinoCatalog> catalogs = catalogConnectorManager.getCatalogs();
-    int size = catalogs.size();
+    List<GravitinoCatalog> gravitinoCatalogs = new ArrayList<>();
+    catalogConnectorManager
```

Comment:
plz add comment to explain why can not call `catalogConnectorManager.getCatalogs()` directly

Reply:
We don't call the `catalogConnectorManager.getCatalogs()`, explain that is more strange

Reply:
However, it was because you called this method before that caused the bug, or you should add some comments for these changes.

Reply:
We have already explained the cause of this bug in the issue. Explaining in the code `why not` calling is more strange.